pkg/issuer/acme/issue.go

```diff
@@ -22,6 +23,9 @@ import (
 const (
 	errorIssueError = "IssueError"
 
+	// default certificateDuration of zero, we let ACME decide
```

Comment:
This const doesn't seem to be used in issue.go, and only in acme.go. Can you move it into acme.go?

Reply:
also rename to `defaultCertificateDuration`